Sources/Recoverable/RecoverableViewState.swift

```diff
@@ -8,22 +8,40 @@
 
 import UIKit
 
-struct RecoverableViewState {
+class RecoverableViewState {
     var backgroundColor: UIColor?
     var cornerRadius: CGFloat
     var clipToBounds: Bool
     
-    // UI text
-    var text: String?
-    
-    // UI image
-    var image: UIImage?
-}
-
-extension RecoverableViewState {
     init(view: UIView) {
         self.backgroundColor = view.backgroundColor
         self.clipToBounds = view.layer.masksToBounds
         self.cornerRadius = view.layer.cornerRadius
     }
 }
+
+class RecoverableTextViewState: RecoverableViewState {
```

Comment:
This should be a struct and not extend `RecoverableViewState`. We don't need the base `RecoverableViewState` model in `UILabel` or `UITextView`.